synnax/pkg/distribution/ontology/schema/resource.go

```diff
@@ -50,12 +51,17 @@ func (id ID) Validate() error {
 	return nil
 }
 
+func (id ID) ParsedKey() (uuid.UUID, error) {
```

Comment:
This method should not be here. Not all ontology IDs are UUIDs (channels are a good example). This is service specific code int the ontology.